tests/com/goide/inspections/GoHighlightingTest.java

```diff
@@ -373,6 +374,21 @@ public VirtualFile compute() throws Throwable {
     myFixture.checkHighlighting();
   }
 
+  public void testUnconventionalStructTag() throws Throwable {
+    myFixture.configureByText("a.go", "package demo; type demoBrokenTags struct {\n"
+                                      + "    CorrectMultipleTags string `json:\"demo,something,something\" dark:\"side\"`\n"
+                                      + "    CorrectSingleTag    string `json:\"demo,something,something\"`\n"
+                                      + "    WrongTag1           string <warning>`json: \"demo\"`</warning>\n"
+                                      + "    WrongTag2           string <warning>`json:\"demo`</warning>\n"
+                                      + "    WrongTag3           string <warning>`json:demo`</warning>\n"
+                                      + "    WrongTag4           string <warning>` json:demo`</warning>\n"
+                                      + "    WrongTag5           string <warning>\"json:demo\"</warning>\n"
+                                      + "    WrongTag6           string <warning>\"json:'demo'\"</warning>\n"
+                                      + "    WrongTag7           string <warning>`json:\"demo\"; bson:\"demo\"`</warning>\n"
+                                      + "}");
+    myFixture.checkHighlighting();
+  }
+
```

Comment:
Please move this code snippet to the separate file in testData/highlighting folder.